traffic-control-openshift/README.md

```diff
@@ -1,29 +1,65 @@
 # Traffic Control on OpenShift
 This demo demonstrates how to use traffic control to test applications running in [OpenShift](https://www.openshift.com/). It uses OpenShift version 3 to run the Kubernetes cluster, [tcd](https://github.com/kinvolk/tcd) to do traffic control on the Kubernetes nodes and [Weave Scope](https://github.com/weaveworks/scope) to monitor, visualize and interact with the Kubernetes cluster.
 
-## What do you need to start?
-- Get an [OpenShift ready VM](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo. You will need to reboot the VM after the first `vagrant up` in order to load the right kernel version and to load the modules needed by tcd.
-- Get the [demo](https://github.com/kinvolk/demo) repository
+To find more information about traffic control and this demo check our [blog post](https://kinvolk.io/blog/2016/05/testing-web-services-with-traffic-control-on-kubernetes/).
 
-You can reach the OpenShift dashboard by pointing your browser to [https://10.2.2.2:8443](https://10.2.2.2:8443). Use user 'admin' and password 'admin' to login.
+### What do you need to start?
+- Get an [OpenShift ready VM](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo.
+- Get the [demo](https://github.com/kinvolk/demo) repository.
 
-For convenience, you can add this to your ~/.ssh/config the output of ```vagrant ssh-config``` and add the following lines.
+### Get the VM
+```
+git clone https://github.com/kinvolk/openshift-evangelists-vagrant-origin.git
+cd openshift-evangelists-vagrant-origin
+vagrant up
+```
+The building process could take several minutes depending of your machines because it is installing additional packages on top of a Fedora base image, and then fetching the last openshift sources and then build it. The final output should look like this:
+
+```
+==> default: You can now access OpenShift console on: https://10.2.2.2:8443/console
+==> default: To use OpenShift CLI, run:
+==> default: $ vagrant ssh
+==> default: $ sudo -i
+==> default: $ oc status
+==> default: $ oc whoami
+==> default: If you have the oc client library on your host, you can also login from your host.
+==> default: $ oc login https://10.2.2.2:8443
+```
+To reboot you can do
+```
+vagrant halt; vagrant up
+```
+You need to reboot the VM after the first `vagrant up` in order to load the right kernel version and to load the modules needed by tcd.
+
+For convenience, you can add this to your ~/.ssh/config the output of `vagrant ssh-config` using as host name `openshift` and add the following lines.
```

Comment:
s/this//

For convenience, you can add the output of `vagrant ssh-config` to your `~/.ssh/config` using as host name `openshift` and add the following lines.

/cc @blixtra